Python/Product/Profiling/Profiling/ProfiledProcess.cs

```diff
@@ -82,6 +89,34 @@ sealed class ProfiledProcess : IDisposable {
             _process.StartInfo = processInfo;
         }
 
+        public void getPyArch(string exe, string arg) {
+            var process = new Process {
+                StartInfo = new ProcessStartInfo {
+                    FileName = exe,
+                    Arguments = arg,
+                    UseShellExecute = false,
+                    RedirectStandardOutput = true,
+                    RedirectStandardError = true,
+                    CreateNoWindow = true
+                },
+                EnableRaisingEvents = true
+            };
+
+            process.ErrorDataReceived += Process_OutputDataReceived;
```

Comment:
If you're not going to be doing anything complicated with the output, you can simplify the handlers by adding them inline, without the need for the callback. Like this:

```csharp
process.ErrorDataReceived += (s, e) => pyArch = e.Data;
process.OutputDataReceived += (s, e) => pyArch = e.Data;
```

Up to you :)

Reply:
@zooba I believe you wanted us to determine arch in a different way. I had Stella just move ahead with what she had so she wouldn't be blocked. Can you please let us know how you would prefer arch to be checked in getArch.py? Thanks 😄